aws/data_source_aws_imagebuilder_image_recipe_test.go

```diff
@@ -59,6 +59,7 @@ resource "aws_imagebuilder_component" "test" {
     }]
     schemaVersion = 1.0
   })
+  working_directory = "/tmp"
```

Comment:
Per the [Contributing Guide](https://github.com/hashicorp/terraform-provider-aws/blob/master/docs/contributing/contribution-checklists.md#enhancementbugfix-to-a-resource), it is preferred to have new optional arguments in new, separate acceptance tests. Will followup these commits adding a resource test and refactoring this data source test.